index.html

```diff
@@ -4,42 +4,175 @@
+        <ul class="grid-container2">
+            <li class="stack2">html</li>
+            <li class="stack2">bootstrap</li>
+            <li class="stack2">Ruby</li>
+        </ul>
+        <a href="#" class="button">See Project</a>
+    </div>
+
+    <div class="third">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
+    </div>
+
+    <div class="fourth">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
+    </div>
+
+    <div class="fifth">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
     </div>
 
+    <div class="sixth">
+        <h2 class="proArt"> Profesional Art Printing Data</h2>
+        <p class="story2">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</p>
+        <div class="grid-container2">
+            <div class="stack2">html</div>
+            <div class="stack2">bootstrap</div>
+            <div class="stack2">Ruby</div>
+        </div>
+        <a href="#" class="button">See Project</a>
+    </div>
 </section>
 
+<section class="item-3">
+    <div class="about">
+
+        <h2 class="aboutMe">About me</h2>
+        <p class="decription">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a class="seeProject">Get My Resume</a>
+
+        <img class="bottomline" src="assets/buttomline.png" alt="border line">
+
+        <div class="language">
+            <div class="triangle">
+                <img class="rect" src="assets/Rectangle%2055.png" alt="rentangle">
+                <h2 class="lang">Language</h2>
```

Comment:
- [ ] Please enclose your **Language**, **Framework** and **Skills** categories inside `ul` tags.

![langs](https://user-images.githubusercontent.com/36542986/134580661-6e23ccae-10ad-4e62-a1aa-f4f245dbe511.jpg)